k8s/namespaces/cpo/case-payment-orders-api/perftest.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: helm.fluxcd.io/v1
```

Comment:
Need to add to kustomize too?

Reply:
it's there in the common place https://github.com/hmcts/cnp-flux-config/blob/cpo_perftest/k8s/namespaces/cpo/kustomization.yaml

Reply:
Sorry, I meant in the common-overlay - e.g. https://github.com/hmcts/cnp-flux-config/blob/cpo_perftest/k8s/demo/common-overlay/cpo/kustomization.yaml

Reply:
aha.. you're saying entry of case-payment-orders-api.yaml in the kustomize. But it works without that for other envs 🤔

Reply:
added now. Thanks @danlysiak